src/main/webapp/view/transcript/transcript.js

```diff
@@ -134,6 +140,20 @@ function removeBold(transcriptLineLiElement) {
   transcriptLineLiElement.classList.remove(BOLD_FONT_WEIGHT);
 }
 
+/** Checks if `transcriptLineLiElement` is bolded. */
```

Comment:
nit: Reword this to: Returns `true` if ...

The statement: Checks does not imply it returns anything.

Reply:
gotcha, done!